app/hotels/src/appRegistry/SingleHotelStandalonePackage.js

```diff
@@ -45,6 +46,8 @@ class SingleHotelStandAlonePackage extends React.Component<Props> {
         roomsConfiguration={this.props.roomsConfiguration}
         hotelId={this.props.hotelId}
         apiProvider="booking" // Not sure if this is in use by native team, but for now, just support booking.com
```

Comment:
I know it's not part of this PR, but the comment seems to indicate this is not true anymore?

Reply:
No, this comment is definitely true. I have never seen this feature been used in the app, I asked if they are using it, but no reply. I Don't really think it is in use.